content/departments/product-engineering/product/process/pricing.md

```diff
@@ -55,3 +55,35 @@ This section is to be determined; in the meantime please work with Product Marke
 ## Choosing the price for add-on features
 
 This section is to be determined; in the meantime please work with Product Marketing and Product Leadership to determine the price for new features that will be charged as add-ons.
+
+## How pricing and packaging decisions are made
```

Comment:
I think there is also a gap around "what is the current strategy of the pricing council". That can be an internal doc for sure, but we should have something that people who are about to work with the pricing council can read in order to have some framework for making their proposal. It would include things like what are tiers are and why, what goes in what tiers, maybe how tiers are enforced, that sort of thing.